bee-tangle/src/metadata.rs

```diff
@@ -83,35 +82,27 @@ impl MessageMetadata {
         self.solidification_timestamp
     }
 
-    pub fn cone_index(&self) -> Option<MilestoneIndex> {
-        self.cone_index
-    }
-
-    pub fn set_cone_index(&mut self, cone_index: MilestoneIndex) {
-        self.cone_index = Some(cone_index);
-    }
-
-    pub fn otrsi(&self) -> Option<MilestoneIndex> {
+    pub fn otrsi(&self) -> Option<IndexId> {
         self.otrsi
     }
 
-    pub fn set_otrsi(&mut self, otrsi: MilestoneIndex) {
+    pub fn set_otrsi(&mut self, otrsi: IndexId) {
         self.otrsi = Some(otrsi);
     }
 
-    pub fn ytrsi(&self) -> Option<MilestoneIndex> {
+    pub fn ytrsi(&self) -> Option<IndexId> {
         self.ytrsi
     }
 
-    pub fn set_ytrsi(&mut self, ytrsi: MilestoneIndex) {
+    pub fn set_ytrsi(&mut self, ytrsi: IndexId) {
         self.ytrsi = Some(ytrsi);
     }
 
     pub fn confirmation_timestamp(&self) -> u64 {
         self.confirmation_timestamp
     }
 
-    pub fn solidify(&mut self) {
+    pub fn set_solid(&mut self) {
```

Comment:
Not a big fan of the renaming, `set_*` assumes a parameter (like the other methods). This one doesn't have a parameter and does a bit more than just setting the flag.

Reply:
I though I did check on what `solidify` was actually doing, and it looked to me like it's just setting booleans and flags.

Reply:
changed